Introduction.tex

```diff
@@ -2,6 +2,8 @@
 
 \begin{document}
 \section{Introduction}
+Simple finite elements in Python (SfePy) uses finite element methods to solve coupled partial differential equation (PDE) in systems up to three dimensions. SfePy is a powerful software that allows complex physical problems to be coded quickly and easily. It has been used successfully in a variety of disciplines, ranging from biomechanical modelling \cite{biomedapplication} to the computational analysis of acoustic transmission coefficients \cite{AcousticTransmission}.\\
+\\In this report, the input file to the SfePy software is a microstructural image which must first be 'cleaned' through segmentation, mesh generation and noise reduction. It can then be imported into the software as a mesh file, where boundary and initial conditions are applied. Fields are then created which can be used to define variables which may be 'unknown field', 'test field' or 'parameter field' \cite{FEMinSfePy} and the material properties are defined.
```

Comment:
Noise reduction was something I just added in the brief for clarity, it may not be used in our project